StatusStrip breaking change (#44043)
dotnet/docs: docs/core/compatibility/windows-forms/9.0/statusstrip-renderer.md

---
title: "Breaking change: System.Windows.Forms.StatusStrip uses a different default renderer"
description: Learn about the breaking change in .NET 9 for Windows Forms where System.Windows.Forms.StatusStrip uses a different default value for the RenderMode property.
ms.date: 12/19/2024
---
# System.Windows.Forms.StatusStrip uses a different default renderer

<xref:System.Windows.Forms.StatusStrip?displayProperty=nameWithType> has been updated to use the default renderer.

## Version introduced

.NET 9

## Previous behavior

Previously, the `StatusStrip`'s `RenderMode` property was set to <xref:System.Windows.Forms.ToolStripRenderMode.System?displayProperty=nameWithType> by default.

## New behavior

> <xref:System.Windows.Forms.StatusStrip?displayProperty=nameWithType> uses the default renderer. Minor changes to the appearance of the `StatusStrip` may be observed.

Reviewer: ```suggestion
<xref:System.Windows.Forms.StatusStrip?displayProperty=nameWithType> uses the default renderer. You might observe minor changes to the appearance of the `StatusStrip`.
```
